python/libcuml/pyproject.toml

```diff
@@ -59,7 +59,7 @@ select = [
 ]
 
 # detect when package size grows significantly
-max_allowed_size_compressed = '500M'
+max_allowed_size_compressed = '550M'
```

Comment:
```suggestion
max_allowed_size_compressed = '525M'
```

I'll approve this as is, but what do you think about a 525M cap?

Reply:
Yes, can do actually. A stricter cap will help us triage situations like today.